plugin_runner/sandbox.py

```diff
@@ -23,92 +26,172 @@
 from RestrictedPython.transformer import (
     ALLOWED_FUNC_NAMES,
     FORBIDDEN_FUNC_NAMES,
+    INSPECT_ATTRIBUTES,
     copy_locations,
 )
+from tini import defaultdict
```

Comment:
I'm not sure we can trust software from this author

Reply:
😆 good catch, auto-import I'm sure